<?xml version="1.0"?>
<launch>

  <!-- The namespace on which the robot's controller managed will be spawned -->
  <arg name="namespace" default="/"/>

  <!-- Load controller settings -->
  <rosparam file="$(find c4g_hardware_interface)/config/c4g_controllers.yaml" command="load" subst_value="true"/>

  <!-- Spawn joint_state_controller -->
  <node name="joint_state_controller_spawner" pkg="controller_manager" type="spawner" args="joint_state_controller"
    respawn="false" ns="$(arg namespace)" output="screen"/>

</launch>